Recurly.Tests/PagerTest.cs

```diff
@@ -102,6 +103,36 @@ public void EnumerablePagesTest()
             Assert.Equal(5, total);
         }
 
+        [Fact]
+        public void PagerFirstTest()
+        {
+            Dictionary<string, object> queryParams = new Dictionary<string, object> {
```

Comment:
I think the left-hand-side can use `var`. Unlike Java, csharp should be smart enough to know the left-hand side type given the right hand's declaration.